README.md

```diff
@@ -104,16 +112,24 @@ products in the given destination `<path>`, creating a `SDT` subfolder:
 ```
 	make clean
 ```
+4. To uninstall one or more products, run the corresponding command.
+Please provide the same `<path>` specified at installation (the parent
+folder of the SDT folder)
+```
+	make uninstall_max DSTDIR=<path>
+	make uninstall_pd DSTDIR=<path> (only for 32 bit)
+	make uninstall_core DSTDIR=<path>
+```
 
 #### Linux
 1. In a terminal, type the following commands:
 ```
 	cd build/linux
 	make
 ```
-2. Install one or more products: By default, the building environment will
-install the SDT shared core library in `/usr/lib` and the Pd *library* in
-`/usr/lib/pd/extras/SDT`. Root privileges may be required to access the
+2. Install one or more products. By default, the building environment will
+install the SDT shared core library in `/usr` and the Pd *library* in
```

Comment:
Perché qui scrivi /usr e non /usr/lib? Vedo che sotto aggiungi /lib al path DSTDIR. Si da per scontato che vada in sottodir lib?

Reply:
Scriverei "core library in `/usr/lib`, headers in `/usr/include`, and ..." O non ho capito?

Reply:
Era proprio perché non viene installato solo in `/usr/lib`, ma anche in `/usr/include`. L'ho messo in quel modo per evidenziare che di default `DSTDIR=/usr` per la core library e `DSTDIR=/usr/lib/pd/extras` per Pd. Provo a mettere tutto per esteo

Reply:
2bd7cb64b50f6641ab6a224e433406998a136ae3